src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
What is the problem with having this as part of the SDK? Injecting everything into the plugin window feels wrong. As discussed already, I even would only inject the dispatcher as I don't see that much value in providing/injecting all the objects.

With all these objects injected, developing a plugin means that all of the stuff you're working with is probably marked as  "Unresolved Variable" and there is no way the IDE could support the developer. Kinda feels like back in the day, when we were relying on global loaded libraries.